jlh/JonFirstApp/app/src/main/res/layout-land/fragment_layout.xml

```diff
@@ -4,17 +4,17 @@
     android:layout_height="match_parent">
     <fragment android:name="com.e_conomic.jonfirstapp.MainFragment"
         android:id="@+id/main_fragment"
-        android:layout_weight="0.5"
         android:layout_width="match_parent"
-        android:layout_height="match_parent">
+        android:layout_height="match_parent"
+        android:layout_weight="0.5">
 
     </fragment>
-    <fragment android:name="com.e_conomic.jonfirstapp.DisplayMessageFragment"
-        android:id="@+id/displaymessage_fragment"
-        android:layout_weight="0.5"
-        android:layout_width="match_parent"
-        android:layout_height="match_parent">
+    <FrameLayout
+        android:id="@+id/fragment_container"
+        android:layout_width="0sp"
```

Comment:
`sp` should be used mostly for text. When setting view heights/widths we should use `dp`. Please check out [this link](http://stackoverflow.com/a/2025541).